event/listener.php

```diff
@@ -62,6 +63,20 @@ public static function getSubscribedEvents()
 	}
 
 	/**
+	 * Adds the permission to the right permission category
+	 *
+	 * @param object $event The event object
+	 * @return void
+	 */
+	public function adv_polls_permissions($event)
```

Comment:
I am not quite informed, so just a question:
Why is this method needed? To add the permission to the right category? Isn't it be already added to some category by default?